content/pt/blue_green_deployment.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Blue Green Deployment
+status: Completed
+category: concept
+---
+
+## O que é
+
+O Blue-green deployment é uma estratégia para atualizar os sistemas de computador em execução com o mínimo de tempo de inatividade. O operador mantém dois ambientes, chamados "blue" e "green".
+Um atende ao tráfego de produção (a versão que os usuários estão usando atualmente), enquanto o outro é atualizado. Depois que o teste é concluído no ambiente inativo (green), o tráfego de produção é alternado (geralmente com o uso de um baleanceador de carga - load balancer). Observe que blue-green deployment geralmente significa alternar os ambientes por completo, abrangendo muitos serviços, todos de uma vez. Surpreendentemente, às vezes o termo é usado em relação a serviços individuais em um sistema. Para evitar essa ambiguidade, o termo "zero-downtime deployment" é preferido quando se refere a componentes individuais.
```

Comment:
**Sugestion**: "Um atende ao tráfego de produção (a versão que os usuários estão usando atualmente), enquanto o outro é atualizado. Depois que o teste é concluído no ambiente inativo (green), o tráfego de produção é **redirecionado** (geralmente com o uso de um baleanceador de carga - load balancer). Observe que blue-green deployment geralmente significa alternar os ambientes por completo, abrangendo muitos serviços, todos de uma vez. **Confusamente**, às vezes o termo é usado em relação a serviços individuais em um sistema. Para evitar essa ambiguidade, o termo "zero-downtime deployment" é preferido quando se refere a componentes individuais."

**Why**: 
"Alternado" could be ambiguous for active balancing between both of them. (it's not the case for blue green).
Original: "(...)production traffic is switched over(...)"

"Confusingly" could be direct translated to "Confusamente" instead of "Surpreendentemente" because it's mentioning when someone could misunderstood the strategy.
Original: Confusingly, sometimes the term is used with regard to individual services within a system. To avoid this ambiguity, the term “zero-downtime deployment” is preferred when referring to individual components.

Reply:
sounds much better, will change it :)